encord/objects/common.py

```diff
@@ -49,8 +43,9 @@ class Attribute(ABC):
     that are part of an :class:`encord.objects.ontology_object.Object`.
     """
 
```

Comment:
I think enum here is stronger type?
You know before checking implementation what is the interface output (one of enum values).

Reply:
Don't think it matters, as this string is only used once for serialisation